source_gen/lib/src/generator_for_annotation.dart

```diff
@@ -123,4 +138,27 @@ abstract class GeneratorForAnnotation<T> extends Generator {
     ConstantReader annotation,
     BuildStep buildStep,
   ) {}
+
+  /// Implement to return source code to generate for [directive].
```

Comment:
It might be good to explain what "directive"s are here, maybe just add (imports, exports, parts).

Reply:
Done,